src/main/java/org/sagebionetworks/WorkflowOrchestrator.java

```diff
@@ -188,16 +189,17 @@ public WorkflowOrchestrator(SynapseClient synapse,
 			String urlString = efh.getExternalURL();
 			URL url = new URL(urlString);
 			// get annotation for the CWL entry point.  Does the file exist?
-			Annotations annotations = synapse.getAnnotations(entityId);
-			Map<String, List<String>> annotationsMap = annotations.getStringAnnotations();
-			if (annotationsMap==null) throw 
-				new IllegalStateException("Expected string annotation called "+
-					ROOT_TEMPLATE_ANNOTATION_NAME+" for "+entityId+" but the entity has no string annotations.");
-			List<String> stringAnnotations = annotationsMap.get(ROOT_TEMPLATE_ANNOTATION_NAME);
-			if (stringAnnotations==null || stringAnnotations.isEmpty()) throw 
-				new IllegalStateException(entityId+" has no string annotation called "+ROOT_TEMPLATE_ANNOTATION_NAME);
-			String rootTemplateString = stringAnnotations.get(0);
-			result.put(evaluationId, new WorkflowURLEntrypointAndSynapseRef(url, rootTemplateString, entityId));
+
+			Annotations annotations = synapse.getAnnotationsV2(entityId);
```

Comment:
It is too early to change it as we have existing data that still uses the old annotations! An alternative is to parameterize the orchestrator to ask which version of the annotations to use.